cyctl/cmd/apply.go

```diff
@@ -161,5 +201,6 @@ var applyCmd = &cobra.Command{
 
 func init() {
 	applyCmd.Flags().StringP("version", "v", "main", "specify cyclops version")
+	applyCmd.Flags().BoolP("telemetry", "t", false, "disable telemetry")
```

Comment:
Can we rename the flag to `disable-telemetry` and update the usage string to something like `disable emitting telemetry metrics from cyclops controller`